share/cmake/utils/CheckSupportAVX.cmake

```diff
@@ -0,0 +1,60 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright Contributors to the OpenColorIO Project.
+
+set(_cmake_cxx_flags_orig "${CMAKE_CXX_FLAGS}")
+
+if(APPLE AND "${CMAKE_OSX_ARCHITECTURES}" MATCHES "arm64;x86_64" 
```

Comment:
Add parenthesis after the `AND` to make it clearer (same applies if the other check* modules)?